dagger.gradle

```diff
@@ -1,178 +0,0 @@
-// Manages classpath and IDE annotation processing config for dagger.
```

Comment:
cc @cfieber no longer needed, but.. may be resurrected if another annotation processor is added, such as auto-value